serverless.yml

```diff
@@ -475,7 +475,25 @@ functions:
       - schedule:
           name: ${self:service}-${self:custom.stage}-index-apy-snapshots
           description: 'index data for vault yields'
-          rate: rate(30 minutes)
+          rate: rate(15 minutes)
+
+  index-vault-definitions:
```

Comment:
this is just moving them to the top since i play with these a lot locally while testing